### ISSUE TITLE
Fetch all button

### ISSUE THREAD

**Issue by NN---** [ForkIssues/TrackerWin#414]
Make fetch to be drop down like GitExtensions does:
![image](https://user-images.githubusercontent.com/580536/64059618-a134ca80-cbc8-11e9-9620-2784f1c6994c.png)

**Comment by Novack** [comment]
Yes this, and let the main button current action be the last used fetch option.

**Comment by ocornut** [comment]
I'd like to add an extra comment on this:

When right-clicking on any remote there is a "Fetch" interface which is a little odd:

There is a "Fetch XXXX" menu entry:

![image](https://user-images.githubusercontent.com/8225057/75871225-86a80c00-5e0c-11ea-9896-b3fe9efbcdf5.png)

Leading us to a dialog that has a "[ ] Fetch all remote" check-box to toggle the behavior.

![image](https://user-images.githubusercontent.com/8225057/75871262-9162a100-5e0c-11ea-8b3d-8856c451fc79.png)

It seems like rather odd UX to me, I would suggest making the context menu show both "Fetch XXXX" and "Fetch All Remotes" entries, and keep the dialog with [X] Fetch all remote removed, and expose the "[ ] Prune Remote Branch. The extra dialog allows to avoid miss-triggering "Fetch all remotes" when not necessary, but you can still perform the action by just pressing Enter by default.

At the moment because "[X] Fetch all remotes" is a saved option, I frequently end up fetching too little or too much.

**Comment by ocornut** [comment]
>  I would suggest making the context menu show both "Fetch XXXX" and "Fetch All Remotes" entries, 

Addendum to previous comment:
- I feel like right-click on "Remotes" root item should rightly expose a "Fetch all remotes" menu item.
- And right-clicking an individual remote can only expose "Fetch remote" and not have a "[X] Fetch all remotes" in the popup.
